week-5/count_file_words/src/main/java/job_hacker/exercise/App.java

```diff
@@ -0,0 +1,31 @@
+package job_hacker.exercise;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class App {
+    private static final String FILE_NAME = "/Lincoln.txt";
+    public static void main(String[] args) {
+        try {
+            File file = new File(App.class.getResource(FILE_NAME).getFile());
+
+            try(Scanner scanner = new Scanner(file)) {
+                int wordCount = 0;
+
+                while (scanner.hasNext()) {
+                    scanner.next();
```

Comment:
this will also count the dashes and commas in texxt and what if there also special characters